bl/policy/policy_factory.go

```diff
@@ -102,6 +123,27 @@ func populateRules(policyRules []cliClient.Rule, customRules []*cliClient.Custom
 	return rules, nil
 }
 
+func checkRuleSchemaExist(customRule *cliClient.CustomRule, identifier string) error {
+	if customRule.Schema == nil && customRule.JsonSchema == "" {
```

Comment:
this should fail in the schema validation step, no need for a special check beforehand